cf-rd/src/main/java/org/eclipse/californium/tools/resources/RDNodeResource.java

```diff
@@ -38,11 +38,15 @@
 	private static final Logger LOGGER = Logger.getLogger(RDNodeResource.class.getCanonicalName());
 	
 	/*
+	 * Use one timer for keeping track of all RDNodeResource's lifeTime expiry tasks.
+	 */
+	private static final Timer lifetimeTimer = new Timer();
```

Comment:
Of course, no problem. Simply close this one and create a new one :-) Again, thanks for contributing...